app/controllers/project/index.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+
+const { Controller, computed, inject: { service } } = Ember;
+
+export default Controller.extend({
+  store: service(),
+  currentUser: service(),
+
+  user: computed.alias('currentUser.user'),
+
+  actions: {
+    saveSubscription(amount) {
+      let project = this.get('project');
+      let queryParams = { amount };
+      this.transitionToRoute('project.donate', project, { queryParams });
```

Comment:
Do we need `queryParams` to be in a hash? Isn't it already a hash with `amount`?  This would send it as:
```
{
  queryParams: {
    amount: x
  }
}
```